### DISCUSSION TITLE
Trailing slash is no more working in 0.23

### DISCUSSION THREAD

**Discussion by GhaziTriki** [pocketbase/pocketbase#5937] [Q&A] [answered]
Previously in 0.22 it was possible to fetch records using a trailing slash `.../records/?filter=...`. With the arrival if 0.23 it no more possible and should update the code with `.../records?filter=...`.

Not sure how many APIs are affected and if this change was intentional, however it should be mentioned in the breaking changes of release notes.

FYI: I am using CURL with bash scripting.

**Comment by ganigeorgiev** [comment, marked as answer]
It is intentional and it is caused by the switch to the Go `net/http.ServeMux` (_technically it is possible to add support for it again but we'll need to either duplicate the route definition or add an extra middleware that for the majority of time will perform unnecessary string operations_).

I'll consider mentioning it in the release notes under the Web APIs changes.

**Reply by GhaziTriki** [reply]
I think it is better to keep it as-is without the trailing slash. The reverse proxy should be able to take care of it. SDKs normally do it. Remains other clients that should updated. Thanks @ganigeorgiev
